scipy/ndimage/interpolation.py

```diff
@@ -339,6 +339,20 @@ def affine_transform(input, matrix, offset=0.0, output_shape=None,
     outside the boundaries of the input are filled according to the given
     mode.
 
+    Given an output image pixel index vector ``o``, the pixel value
+    is determined from the input image at position ``np.dot(matrix,o) + offset``.
+
+    A diagonal matrix can be specified by supplying a one-dimensional
+    array-like to the matrix parameter, in which case a more efficient
+    algorithm is applied.
+
+    .. versionchanged:: 0.18.0
```

Comment:
I actually don't know... Do you think it is the `.. versionchanged::` that makes it fail? I found that in the spinx documentation, but I really don't know sphinx nor scipy's documentation configuration at all... I'll try to investigate locally.